driver/BUILD.bazel

```diff
@@ -129,11 +129,16 @@ cc_binary(
         "//agent:jazzer_agent_deploy.jar",
     ],
     linkopts = [
-        "-fsanitize=address",
-        "-static-libsan",
     ] + select({
-        "@platforms//os:windows": [],
+        "@platforms//os:windows": [
+            # Sanitizer runtimes have to be linked manually on Windows:
+            # https://devblogs.microsoft.com/cppblog/addresssanitizer-asan-for-windows-with-msvc/
+            "/wholearchive:clang_rt.asan-x86_64.lib",
+            "/wholearchive:clang_rt.asan_cxx-x86_64.lib",
+        ],
         "//conditions:default": [
+            "-fsanitize=address",
+            "-static-libsan",
```

Comment:
isn't this the default behavior for clang on Linux?

Reply:
It is on Linux, but not on macOS. Since it may change on Linux in the future (albeit unlikely), I found having this here explicitly to be safer.

Reply:
agreed